.travis.yml

```diff
@@ -0,0 +1,28 @@
+language: java
+before_script:
+  - unset _JAVA_OPTIONS
```

Comment:
Looks like this step is to workaround an issue in travis. The issue can cause random problems in the test. Consider whether it is really necessary for our case, if yes, perhaps would be good to add a comment or doc to explain.

Reply:
Ok